docs/reference/search/search-your-data/semantic-search.asciidoc

```diff
@@ -20,95 +20,57 @@ image::images/search/vector-search-oversimplification.png[A simplified represent
 
-include::{es-ref-dir}/tab-widgets/semantic-search/field-mappings-widget.asciidoc[]
 
 [discrete]
-[[semantic-search-generate-embeddings]]
-=== Generate text embeddings
+[[using-query]]
+=== Using the right query
 
-Once you have created the mappings for the index, you can generate text embeddings from your input text.
-This can be done by using an
-<<ingest,ingest pipeline>> with an <<inference-processor,inference processor>>.
-The ingest pipeline processes the input data and indexes it into the destination index.
-At index time, the inference ingest processor uses the trained model to infer against the data ingested through the pipeline.
-After you created the ingest pipeline with the inference processor, you can ingest your data through it to generate the model output.
+Crafting the right query is crucial for semantic search.
+The query type you should use depends first on whether you are using the recommended `semantic_text` workflow.
+If not, it depends on the vector type in which your embeddings are stored.
 
-include::{es-ref-dir}/tab-widgets/semantic-search/generate-embeddings-widget.asciidoc[]
+[cols="3*", options="header"]
+|=======================================================================================================================================================================================================
+| Field to query                    | Query to use                                      | Notes                                                                                                                                                             
+| <<semantic-text,`semantic_text`>> | <<query-dsl-semantic-query,`semantic`>> .         | The `semantic_text` field handles generating embeddings for you at index time and query time.                                                                     
+| <<sparse-vector,`sparse_vector`>> | <<query-dsl-sparse-vector-query,`sparse vector`>> | The `sparse_vector` query can generate query embeddings for you, but you can also provide your own. You are expected to provide embeddings at index time.
+| <<dense-vector,`dense_vector`>>   | <<query-dsl-knn-query,`knn`>>                     | The `knn` query can generate query embeddings for you, but you can also provide your own. You are expected to provide embeddings at index time.
+|=======================================================================================================================================================================================================
 
-Now it is time to perform semantic search!
+If you want {es} to generate embeddings for you both index and query time, use the `semantic_text` field and the `semantic` query.
+If you want to bring your own embeddings, store them in {es} and use the `sparse_vector` or `dense_vector` field type and the associated query depending on the NLP model you used for generating the embeddings.
```

Comment:
Would like to see mention of the BYOE use case earlier in this doc somewhere too.